src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,194 @@
+- **Use the ++ctrl+enter++ or ++cmd+enter++ keyboard shortcut**. With a data entry cell selected, use one of these keyboard shortcuts.
+
+The data details sidebar will stay open until you close it so that you can keep it open for reference as you work through a data entry form.
+
+#### Mark data values for follow-up { #aggregate_data_entry_app.mark_followup }
+
+Marking data values for follow-up can be a useful way of highlighting suspicious or odd values that need to be investigated. Data values marked for follow-up will still be saved, but they will be highlighted in the **Data Quality** app for further investigation or analysis.
+
+To mark a data value for follow-up, [make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then click the *Mark for follow-up* button in the top section.
+
+![](resources/images/data_entry_beta/followup-button.png) { width=60% }
+
+To unmark a data value, click the *Unmark for follow-up* button.
+
+![](resources/images/data_entry_beta/followup-unmark-button.png) { width=60% }
+
+#### Comment on a data value { #aggregate_data_entry_app.comments }
+
+You can add a comment to any data value. Comments can be useful to add more information about a value, like noting reasons why a value is unusually high or out of the normal range.
+
+To add a comment to a data value, [make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then click the *Add comment* button in the *Comment* section.
+
+![](resources/images/data_entry_beta/comment-add-button.png) { width=50% }
+
+Once you've written a comment, click the *Save comment* button.
+
+If a data value already has a comment, you can edit it by clicking the *Edit comment* button below the comment.
+
+![](resources/images/data_entry_beta/comment-edit-button.png) { width=50% }
+
+#### Minimum and maximum limits { #aggregate_data_entry_app.limits}
+
+A data value can have a minimum and maximum limit. These limits won't allow someone to enter values outside of the minimum or maximum. 
+
+To add limits to a data value, [make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then click the *Add limits* button in the *Min and max limits* section. If possible, an average value will be shown to help you create reasonable limits.
+
+![](resources/images/data_entry_beta/limits-add-button.png) { width=50% }
+
+Once you've added a minimum and maximum limit, click the *Save limits* button.
+
+If a data value already has limits set up, you can change them or remove them. To change the limits, click the *Edit limits* button. To remove the limits, click the *Delete limits* button.
+
+![](resources/images/data_entry_beta/limits-edit-button.png) { width=50% }
+
+Adding, changing, or removing limits requires the correct user privileges.
```

Comment:
Use note/caution format and add a little info to clarify that it is not simple for the current user (not sure I'm completely happy with my text, though?)
```suggestion
> **Caution**
>
> Adding, changing, or removing limits requires the correct user privileges. When limits are applied to a data value, the same limits apply to everyone using same data set.
```